### PR TITLE
[TAPIUI] Update Polish translation

### PR DESCRIPTION
**Pull request by pithwz**

### CODE REVIEW

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 1]
```diff
@@ -295,32 +296,32 @@ BEGIN
...
-    AUTORADIOBUTTON "Wybierania &Tonowego", 1036, 95, 139, 55, 12, WS_GROUP | NOT WS_TABSTOP
-    AUTORADIOBUTTON "Wybierania &Ciągiem Impulsów", 1037, 155, 139, 55, 12, NOT WS_TABSTOP
+    AUTORADIOBUTTON "Wybierania &Tonowego", 1036, 95, 139, 55, 10, WS_GROUP | NOT WS_TABSTOP
+    AUTORADIOBUTTON "Wybierania &Ciągiem Impulsów", 1037, 207, 139, 115, 10, NOT WS_TABSTOP
 END
 
 500 DIALOGEX 0, 0, 235, 88
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_SETFOREGROUND | DS_CENTER | WS_POPUPWINDOW | WS_CAPTION
-CAPTION "Explorator ReactOS"
+CAPTION "Ekslorator ReactOS"
```
Comment (archeYR) [inline dll/win32/tapiui/lang/pl-PL.rc:324]:
```suggestion
CAPTION "Eksplorator ReactOS"
```

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 2]
```diff
@@ -258,24 +259,24 @@ CAPTION "Zaawansowane"
...
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Wybierz providera, którego chciałbyś zainstalować, z listy poniżej, i naciśnij Dodaj.", 2000, 7, 7, 241, 8
-    LTEXT "&Providerzy usług telefonicznych:", -1, 7, 23, 144, 10
-    LISTBOX 1055, 7, 34, 238, 74, LBS_STANDARD | WS_TABSTOP
-    PUSHBUTTON "&Dodaj", 1005, 141, 106, 50, 14
-    DEFPUSHBUTTON "Anuluj", 2, 195, 106, 50, 14
+    LTEXT "Wybierz dostawcę, którego chcesz zainstalować, z listy poniżej, i naciśnij Dodaj.", 2000, 7, 7, 241, 17
+    LTEXT "&Dostawcy usług telefonicznych:", -1, 7, 30, 144, 10
+    LISTBOX 1055, 7, 41, 238, 74, LBS_STANDARD | WS_TABSTOP
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:277]: Please revert stylistic Y-axis changes, they should be done separately for all languages.

See en-US and other translations: https://github.com/reactos/reactos/blob/7fe7f5e4c107451c4485cc53f140ba5e93612b63/dll/win32/tapiui/lang/en-US.rc#L267

Comment (pithwz) [reply dll/win32/tapiui/lang/pl-PL.rc:277]: if i revert these changes, text is unreadable

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 3]
```diff
@@ -47,12 +48,12 @@ BEGIN
     EDITTEXT 1059, 190, 136, 48, 12, ES_AUTOHSCROLL
     LTEXT "Użyj następującego kodu dla połączeń &międzynarodowych:", -1, 14, 154, 170, 8
     EDITTEXT 1060, 190, 152, 48, 12, ES_AUTOHSCROLL
-    AUTOCHECKBOX "Aby &wyłączyć oczekiwanie przy połączeniu, wybierz:", 1035, 14, 177, 170, 10
+    AUTOCHECKBOX "Aby &wyłączyć oczekiwanie przy połączeniu, wybierz:", 1035, 10, 177, 170, 10
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:51]:
```suggestion
    AUTOCHECKBOX "Kod wyłączenia alertu:", 1035, 14, 177, 170, 10
```
Shorter translation.

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 4]
```diff
@@ -47,12 +48,12 @@ BEGIN
...
     COMBOBOX 1013, 190, 176, 48, 73, CBS_DROPDOWN | CBS_SORT | WS_VSCROLL
-    LTEXT "Wybierz numer:", -1, 14, 195, 40, 8
-    AUTORADIOBUTTON "&Tonowo", 1036, 58, 194, 50, 10, NOT WS_TABSTOP
-    AUTORADIOBUTTON "&Ciągiem impulsów", 1037, 112, 194, 50, 10, NOT WS_TABSTOP
-    LTEXT "Numer telefoniczny będzie wybierany jako:", 1052, 7, 219, 100, 8
+    LTEXT "Wybierz numer:", -1, 10, 195, 40, 8
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:53]:
```suggestion
    LTEXT "Wybierz numer:", -1, 14, 195, 54, 8
```

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 5]
```diff
@@ -193,11 +194,11 @@ BEGIN
...
     CONTROL "List1", 1033, "SysListView32", WS_BORDER | WS_TABSTOP | 0x0000411D, 25, 128, 66, 32, WS_EX_CLIENTEDGE
     PUSHBUTTON "&Dodaj", 1005, 95, 128, 50, 14
-    PUSHBUTTON "&Skasuj", 1024, 95, 146, 50, 14
+    PUSHBUTTON "&Usuń", 1024, 95, 146, 50, 14
     LTEXT "Podczas wybierania numerów, które zawierają powyższe prefiksy:", -1, 14, 184, 224, 8
-    AUTOCHECKBOX "&Wybierz:", 1012, 14, 198, 30, 12
-    EDITTEXT 1007, 46, 196, 42, 14, ES_AUTOHSCROLL
-    AUTOCHECKBOX "Dołącz numer zamiejscowy", 1029, 14, 214, 85, 12
+    AUTOCHECKBOX "&Wybierz:", 1012, 14, 198, 66, 12
+    EDITTEXT 1007, 87, 196, 42, 14, ES_AUTOHSCROLL
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:200]:
```suggestion
    AUTOCHECKBOX "&Wybierz:", 1012, 14, 198, 44, 12
    EDITTEXT 1007, 60, 196, 42, 14, ES_AUTOHSCROLL
```

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 6]
```diff
@@ -258,24 +259,24 @@ CAPTION "Zaawansowane"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON 201, 2000, 7, 7, 20, 20
-    LTEXT "Następujący providerzy usług telefonicznych są zainstalowani na tym komputerze:", -1, 40, 7, 205, 8
-    LTEXT "&Providerzy:", -1, 7, 35, 238, 8
+    LTEXT "Następujący dostawcy usług telefonicznych są zainstalowani na tym komputerze:", -1, 40, 7, 205, 21
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:262]: Some other languages affected by this problem, should be solved separately.
```suggestion
    LTEXT "Następujący dostawcy usług telefonicznych są zainstalowani na tym komputerze:", -1, 40, 7, 205, 8
```

File: dll/win32/tapiui/lang/pl-PL.rc (hunk 2 above)
Comment (binarymaster) [reply dll/win32/tapiui/lang/pl-PL.rc:277]: Alright, this one looks ok.

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 7]
```diff
@@ -295,32 +296,32 @@ BEGIN
     LTEXT "Numer do &linii zewnętrznej", -1, 107, 137, 189, 10
     EDITTEXT 1010, 107, 149, 45, 12, ES_UPPERCASE | ES_AUTOHSCROLL
     LTEXT "System telefoniczny używa:", -1, 107, 168, 189, 10
-    AUTORADIOBUTTON "Wybierania &Tonowego", 1036, 107, 178, 57, 14, WS_GROUP | NOT WS_TABSTOP
-    AUTORADIOBUTTON "Wybierania &Ciągiem Impulsów", 1037, 175, 178, 57, 14, NOT WS_TABSTOP
+    AUTORADIOBUTTON "Wybierania &Tonowego", 1036, 107, 178, 103, 10, WS_GROUP | NOT WS_TABSTOP
+    AUTORADIOBUTTON "Wybierania &Ciągiem Impulsów", 1037, 107, 190, 115, 10, NOT WS_TABSTOP
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:300]:
```suggestion
    AUTORADIOBUTTON "Wybierania &Tonowego", 1036, 107, 178, 110, 10, WS_GROUP | NOT WS_TABSTOP
    AUTORADIOBUTTON "Wybierania &Ciągiem Impulsów", 1037, 107, 190, 110, 10, NOT WS_TABSTOP
```

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 8]
```diff
@@ -295,32 +296,32 @@ BEGIN
...
     DEFPUSHBUTTON "OK", 1, 192, 204, 50, 14
     PUSHBUTTON "Anuluj", 2, 246, 204, 50, 14
 END
 
-400 DIALOGEX 0, 0, 290, 151
+400 DIALOGEX 0, 0, 323, 153
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:305]:
```suggestion
400 DIALOGEX 0, 0, 296, 151
```

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 9]
```diff
@@ -295,32 +296,32 @@ BEGIN
...
 BEGIN
     CONTROL 302, 302, "STATIC", SS_BITMAP, 0, 0, 88, 155
-    LTEXT "Zanim będziesz mógł dokonać jakichkolwiek połączeń telefonicznych albo modemowych, ReactOS porzebuje następujących informacji o twojej obecnej lokalizacji.", 2000, 95, 2, 188, 25
-    LTEXT "&W jakim kraju/regionie teraz jesteś?", -1, 95, 30, 188, 10
-    COMBOBOX 1006, 95, 41, 177, 68, CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL
-    LTEXT "Numer &międzymiastowy", -1, 95, 61, 188, 10
+    LTEXT "Zanim będziesz mógł dokonać jakichkolwiek połączeń telefonicznych albo modemowych, ReactOS porzebuje następujących informacji o twojej obecnej lokalizacji.", 2000, 95, 3, 225, 25
+    LTEXT "&W jakim kraju/regionie teraz jesteś?", -1, 95, 34, 188, 10
+    COMBOBOX 1006, 95, 45, 177, 68, CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL
+    LTEXT "Numer &międzymiastowy", -1, 95, 64, 188, 10
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:313]:
```suggestion
    LTEXT "Zanim będziesz mógł dokonać jakichkolwiek połączeń telefonicznych albo modemowych, ReactOS porzebuje następujących informacji o twojej obecnej lokalizacji.", 2000, 95, 2, 188, 25
    LTEXT "&W jakim kraju/regionie teraz jesteś?", -1, 95, 30, 188, 10
    COMBOBOX 1006, 95, 41, 177, 68, CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL
    LTEXT "Numer &międzymiastowy", -1, 95, 61, 188, 10
```

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 10]
```diff
@@ -295,32 +296,32 @@ BEGIN
...
     EDITTEXT 1034, 95, 73, 40, 14, ES_AUTOHSCROLL
-    LTEXT "Numer do &linii zewnętrznej", -1, 95, 96, 188, 10
-    EDITTEXT 1010, 95, 107, 40, 14, ES_UPPERCASE | ES_AUTOHSCROLL
+    LTEXT "Numer do &linii zewnętrznej", -1, 95, 97, 188, 10
+    EDITTEXT 1010, 95, 109, 40, 14, ES_UPPERCASE | ES_AUTOHSCROLL
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:316]:
```suggestion
    LTEXT "Numer do &linii zewnętrznej", -1, 95, 96, 188, 10
    EDITTEXT 1010, 95, 107, 40, 14, ES_UPPERCASE | ES_AUTOHSCROLL
```

File: dll/win32/tapiui/lang/pl-PL.rc [hunk 11]
```diff
@@ -295,32 +296,32 @@ BEGIN
...
     LTEXT "System telefoniczny używa:", -1, 95, 129, 159, 10
-    AUTORADIOBUTTON "Wybierania &Tonowego", 1036, 95, 139, 55, 12, WS_GROUP | NOT WS_TABSTOP
-    AUTORADIOBUTTON "Wybierania &Ciągiem Impulsów", 1037, 155, 139, 55, 12, NOT WS_TABSTOP
+    AUTORADIOBUTTON "Wybierania &Tonowego", 1036, 95, 139, 55, 10, WS_GROUP | NOT WS_TABSTOP
+    AUTORADIOBUTTON "Wybierania &Ciągiem Impulsów", 1037, 207, 139, 115, 10, NOT WS_TABSTOP
```
Comment (binarymaster) [inline dll/win32/tapiui/lang/pl-PL.rc:319]:
```suggestion
    AUTORADIOBUTTON "Wybierania &Tonowego", 1036, 95, 139, 85, 10, WS_GROUP | NOT WS_TABSTOP
    AUTORADIOBUTTON "Wybierania &Ciągiem Impulsów", 1037, 185, 139, 108, 10, NOT WS_TABSTOP
```

**Pull request merged**